tiledb/cf/engines/__init__.py

```diff
@@ -1,17 +1,86 @@
-from typing import Dict, Optional, Union
+from typing import Dict, Optional, Tuple, Union
+
+import numpy as np
 
 import tiledb
 
-from ..creator import DataspaceConverter
+_DEFAULT_INDEX_DTYPE = np.dtype("uint64")
+
+
+def converter_from_netcdf_file(
```

Comment:
Not sure I see the point in this function and the next one; 60+ LoC for an alias to `NetCDF4ConverterEngine.from_file` and `NetCDF4ConverterEngine.from_group`, why?

Reply:
netCDF4 is an optional dependency, so I'm trying to avoid importing the module directly into __init__